Pear/Controller/CommunityViewController.swift

```diff
@@ -127,29 +119,15 @@ extension CommunityViewController: UITableViewDataSource {
 
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         let user = users[indexPath.row]
-        navigationController?.pushViewController(ProfileViewController(userId: user.netID), animated: true)
+        navigationController?.pushViewController(ProfileViewController(userId: user.id), animated: true)
     }
 }
 
 // MARK: - SearchBarDelegate
 extension CommunityViewController: UISearchBarDelegate {
 
     private func searchUsers(query: String) {
-        NetworkManager.shared.searchUsers(query: query).observe { response in
-            DispatchQueue.main.async {
-                switch response {
-                case .value(let value):
-                        guard value.success else {
-                            print("Network error: could not search users")
-                            return
-                        }
-                        self.users = value.data
-                        self.fadeCommunityTableView.view.reloadData()
-                case .error:
-                    print("Network error: could not search users")
-                }
-            }
-        }
+        // TODO: Fill in with network call
```

Comment:
yup implemented recently as mentioned and remember it's the same route as get all used to be and you can pass in both a query and the custom pagination parameters